src/hooks/screen/useCreateChat.ts

```diff
@@ -36,13 +34,6 @@ const useCreateChat = () => {
       members?.forEach(async (member) => {
         const userMember = UserSchema.fromMemberWebsocketObject(member, response?.channel?.id);
         await userMember.saveOrUpdateIfExists(localDb);
-
-        const memberSchema = ChannelListMemberSchema.fromWebsocketObject(
-          response?.channel?.id,
-          uuid(),
-          member
-        );
-        await memberSchema.save(localDb);
       });
     } catch (e) {
       console.log(e, 'error on memberSchema');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the creation and saving of `memberSchema` objects could potentially lead to issues if other parts of your application rely on these objects being present in the local database. Please ensure that this change does not break any functionality that depends on `ChannelListMemberSchema`.

If the `memberSchema` objects are no longer needed, it would be better to remove the console log statement at line 39 as it references 'error on memberSchema' which might be misleading since `memberSchema` is no longer part of this code block.

```diff
-        console.log(e, 'error on memberSchema');
+        console.log(e, 'error on userMember');
```
This change will make the error message more accurate and less confusing for future debugging.


<!-- This is an auto-generated comment by OSS CodeRabbit -->